performance_manager/lib/rt_trip_updates.py

```diff
@@ -155,18 +160,115 @@ def get_and_unwrap_tu_dataframe(
     events["is_moving"] = False
     events["pk_id"] = None
 
+    return events
+
+
+def join_gtfs_static(
+    trip_updates: pandas.DataFrame, db_manager: DatabaseManager
+) -> pandas.DataFrame:
```

Comment:
new function to get static timestamp value for joining event to gtfs static data

this function is pretty well commented explaining business logic